.github/actions/go-cache/action.yml

```diff
@@ -13,17 +13,14 @@ runs:
       run: echo 'C:\Program Files\Git\usr\bin' >> $GITHUB_PATH
 
     - name: Load Go cache
+      id: load-go-cache
       uses: actions/cache@v4
       with:
         path: |
-          ~/go/pkg
-          ~\go\pkg
-          ~/go/bin
-          ~\go\bin
-          ~/.cache/golangci-lint
-          ~/Library/Caches/golangci-lint
-          ~\AppData\Local\golangci-lint
```

Comment:
As the linter ends up quicker even without caching I adjusted it currently by removing the files. The cache is smaller.

Reply:
Can be reworked with another PR where support for Azure Blob storage will be provided.